src/test/java/seedu/address/logic/parser/ViewCommandParserTest.java

```diff
@@ -24,10 +26,20 @@ public void parse_nameSpecified_success() {
     public void parse_missingCompulsoryField_failure() {
         String expectedMessage = String.format(MESSAGE_INVALID_COMMAND_FORMAT, ViewCommand.MESSAGE_USAGE);
 
+        // no parameters
+        assertParseFailure(parser, ViewCommand.COMMAND_WORD, expectedMessage);
+
         // no name specified
         assertParseFailure(parser, PHONE_DESC_AMY, expectedMessage);
 
         // no index and no field specified
         assertParseFailure(parser, "", expectedMessage);
+
+        // no preamble
+        assertParseFailure(parser, PREFIX_NAME + AMY.getName().fullName, expectedMessage);
```

Comment:
similarly, no command word instead of no preamble. 
Preamble is what is between command word and first parameter